app/code/Magento/SalesSequence/Model/ResourceModel/Profile.php

```diff
@@ -76,4 +77,21 @@ public function loadActiveProfile($metadataId)
         }
         return $profile;
     }
+
+    /**
+     * Get profile ids by metadata ids
+     *
+     * @param int[] $metadataIds
+     * @return int[]
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function getProfileIdsByMetadataIds(array $metadataIds)
```

Comment:
Introduction of a new public method is backwards incompatible and cannot be added to a patch release. Please create a new class for this functionality